packages/adapter-cloudflare/files/worker.js

```diff
@@ -50,12 +50,28 @@ export default {
 
 		// dynamically-generated pages
 		try {
-			return await server.respond(req, {
-				platform: { env, context },
-				getClientAddress() {
-					return req.headers.get('cf-connecting-ip');
+			// @ts-ignore
```

Comment:
Is there some package like https://www.npmjs.com/package/@cloudflare/workers-types that we can use so that we don't have to `@ts-ignore` this?

If there's no solution then it should be turned into `@ts-expect-error` along with a comment explaining why it cannot be addressed

Reply:
Hey @benmccann, I tried to add the package above but it started to throw ~30 errors in the JS files so for now I've used `@ts-expect-error` with a comment as to why.